### PR TITLE
Add CacheObject jcache serialization test

### PR DESCRIPTION
**Pull request by ryanesch**
... for https://github.com/openliberty/open-liberty/issues/20568

### CODE REVIEW

**Comment by ryanesch** [comment]
#build

File: dev/io.openliberty.jcache.internal/test/io/openliberty/jcache/CacheObjectSerializationTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+     * replacing the x with the current version + 1.
+     * Then write a test that deserializes that version and all
+     * previous CacheObject_x.ser files.
+     */
+    //@Test
+    public static void main(String[] args) throws Exception {
+        //Create CacheObject
+        CacheObject object = new CacheObject(null, bytes_cacheobject_1);
+
+        String filename = "test-resources/ser-files/CacheObject_1.ser";
```
Comment (jvanhill) [inline dev/io.openliberty.jcache.internal/test/io/openliberty/jcache/CacheObjectSerializationTest.java]: Assuming that 1 was supposed to x.

```suggestion
        String filename = "test-resources/ser-files/CacheObject_x.ser";
```

**Review by jvanhill — APPROVED** [review]
Approving, but probably should address the 1 comment I had.

**Comment by ryanesch** [comment]
#build

**Comment by ryanesch** [comment]
#run-libby-bot

**Pull request merged**